tasks/main.yml

```diff
@@ -5,111 +5,127 @@
     - "{{ ansible_os_family }}-{{ ansible_distribution_major_version }}.yml"
     - "{{ ansible_os_family }}.yml"
 
-- name: Ensure replication user exists on master.
+- name: import assert.yml
+  import_tasks: assert.yml
+  run_once: yes
+  delegate_to: localhost
+
+- name: Ensure replication user exists on primary.
   mysql_user:
     name: "{{ mysql_replication_user.name }}"
     host: "{{ mysql_replication_user.host | default('%') }}"
     password: "{{ mysql_replication_user.password }}"
     priv: "{{ mysql_replication_user.priv | default('*.*:REPLICATION SLAVE,REPLICATION CLIENT') }}"
     state: present
     login_unix_socket: "{{ mysql_socket }}"
+  #no_log: true
```

Comment:
You forgot the debug comment ;)

Reply:
Oh my bad! Corrected on the last version.